eat_test.go

```diff
@@ -69,6 +79,8 @@ func cborRoundTripper(t *testing.T, tv Eat, expected []byte) {
 func jsonRoundTripper(t *testing.T, tv Eat, expected string) {
 	data, err := tv.ToJSON()
 
+	t.Logf("JSON: '%s'", string(data))
```

Comment:
Are these debug statements left in, or is the intention to leave them?

Reply:
good question.  I'm actually undecided: they are quite useful when something goes bananas and they don't pollute stdout/stderr unless `go test` is invoked with the verbose flag.  So they are not super intrusive, although they are not nice to see...  I cannot convince myself one way or another.  You pick! :-)

Reply:
If they're only visible with verbose flag, I don't see the harm in leaving them in.